packages/ui/src/proposals/components/ProposalStages.tsx

```diff
@@ -3,44 +3,73 @@ import React, { useMemo } from 'react'
 import { ControlProps } from '@/common/components/forms'
 import { CheckboxIcon, CrossIcon } from '@/common/components/icons'
 import { TabContainer, TabsContainer } from '@/common/components/Tabs'
+import { Tooltip } from '@/common/components/Tooltip'
 import { last, repeat } from '@/common/utils'
 import { ProposalStatus, ProposalStatusUpdates } from '@/proposals/types'
 
 export interface ProposalStagesProps extends ControlProps<number> {
-  status: ProposalStatus
+  roundStatus: ProposalStatus
   updates: ProposalStatusUpdates[]
-  constitutionality?: number | string
+  constitutionality?: number
 }
 
-export const ProposalStages = ({ status, updates, constitutionality = '-', value, onChange }: ProposalStagesProps) => {
-  const rounds = useMemo(() => {
+type RoundState = 'approved' | 'rejected' | 'deciding' | 'disabled'
```

Comment:
Maybe these should be included from `types.ts`? I checked the code and there are [multiple locations](https://github.com/search?q=org%3AJoystream+dormant&type=code) that hard-code possible stages. @thesan wdyt? For example [here](https://github.com/Joystream/pioneer/blob/bb6f63bd6455c812b8934d96f0b964bb4192d782/packages/ui/src/proposals/types/proposals.ts#L27):
> type CurrentProposalStatus = Extract<ProposalStatus, 'deciding' | 'gracing' | 'dormant'>

Reply:
This type is just for this file. It's the status of a round which is different from the status of the proposal this is why I think it should stay here and not be exported.